.github/workflows/push.yml

```diff
@@ -26,3 +26,30 @@ jobs:
 
        - name: Perform CodeQL Analysis
          uses: github/codeql-action/analyze@v1
+         
```

Comment:
We don't need to run the code coverage report on pushes. Please revert all changes to this file.